core/src/main/java/contacts/core/data/DataQuery.kt

```diff
@@ -94,7 +94,7 @@ interface DataQueryFactory {
 }
 
 @Suppress("FunctionName")
-internal fun DataQuery(contacts: Contacts, isProfile: Boolean): DataQueryFactory =
+internal fun DataQueryFactory(contacts: Contacts, isProfile: Boolean): DataQueryFactory =
```

Comment:
I noticed that this function did not have the correct name so I corrected it. Just boyscouting =)